gulpfile.js

```diff
@@ -39,7 +39,13 @@ gulp.task('typings', function(done) {
 });
 
 gulp.task('tsc', function(done) {
-  runSpawn(done, 'node_modules/typescript/bin/tsc');
+  var tsc = spawn('node_modules/typescript/bin/tsc');
```

Comment:
Hmmm, there must be a way to do this with stdio options. Gulp must be doing something funny to make `inherit` fail. Can we try `{stdio: [process.stdin, process.stdout, process.stderr]}`?